docs/docs/docs.md

```diff
@@ -40,103 +28,139 @@ final case class GHResponse[A](
 )
 ```
 
+To make HTTP calls, Github4s relies on [an http4s' HTTP client][http4s-client] which needs to be
+supplied as we'll see later. Here, we are making use of `JavaNetClientBuilder` because of its ease
+of use in a REPL. However, for production use you should prefer `BlazeClientBuilder` over it as
+detailed in [the documentation][http4s-client].
+
+```scala mdoc:silent
+import java.util.concurrent.Executors
+
+import cats.effect.{Blocker, ContextShift, IO}
+import org.http4s.client.{Client, JavaNetClientBuilder}
+
+import scala.concurrent.ExecutionContext.global
+
+val httpClient: Client[IO] = {
+  val blockingPool = Executors.newFixedThreadPool(5)
+  val blocker = Blocker.liftExecutorService(blockingPool)
+  implicit val cs: ContextShift[IO] = IO.contextShift(global)
+  JavaNetClientBuilder[IO](blocker).create // use BlazeClientBuilder for production use
+}
+```
+
 As an introductory example, we can get a user with the following:
 
-```scala mdoc:silent:fail
-val user1 = Github[IO](accessToken).users.get("rafaparadela")
+```scala mdoc:silent
+import github4s.Github
+val accessToken = sys.env.get("GITHUB4S_ACCESS_TOKEN")
+val user1 = Github[IO](httpClient, accessToken).users.get("rafaparadela")
 ```
 
 `user1` in this case is a `IO[GHResponse[User]]`.
 
-### Using `F[_]: cats.effect.ConcurrentEffect`
+### Using `F[_]: cats.effect.Sync`
```

Comment:
The actual context bound in the sample code below is `: Async: ContextShift`

Reply:
This is because of the chosen client, if it were `BlazeClientBuilder` it would be `ConcurrentEffect`. Our constraint is actually `Sync`: https://github.com/47degrees/github4s/pull/373/files#diff-9a35e1db598f031cb40b6fcaf7266225R24. Do you think this needs additional information?

Reply:
or I could write the example as:

```scala
  def u1[F[_]: Sync](httpClient: Client[F]): F[GHResponse[User]] =
    Github[F](httpClient, accessToken).users.get("juanpedromoreno")
```

Reply:
which might be closer to what people end up writing anyway